connect/runtime/src/main/java/org/apache/kafka/connect/util/KafkaBasedLog.java

```diff
@@ -237,6 +239,9 @@ public void send(K key, V value, org.apache.kafka.clients.producer.Callback call
         producer.send(new ProducerRecord<>(topic, key, value), callback);
     }
 
+    public int getPartitionCount() {
```

Comment:
nit: traditionally we skip the `get/set` prefixes in method names in this code base (there maybe a few counter examples floating around). 

How about, `partitionCount`?